src/app-config/auth.ts

```diff
@@ -18,24 +18,35 @@ interface AuthResult {
   id_token?: string;
 }
 
+export type AuthClientOptions = {
+  checkLogin: () => Promise<User>;
```

Comment:
So with the last minute realization on Friday. Do we need the checkLogin callbacks? Your initial logic which called alias in the checkLogin seemingly would have been sufficient.

Reply:
hmmm, there's definitely some room for re-working, but we can't just outright get rid of it because we use it to acquire the User object here:

https://github.com/architect-team/architect-cli/blob/bugfix/621/src/app-config/auth.ts#L166

Though there's something weirdly circular about the current implementation of the two callbacks now that you point it out.

Reply:
@tjhiggins updated this. Feel free to review and then let's get it merged in so we capture events correctly.